README.md

```diff
@@ -13,7 +11,7 @@ add flex box and edit them.
 
 add background image and edit.
 
-add button and edit them.
+links from figma
```

Comment:
- [ ] Kindly, make sure your readme file is without grammar errors, like this one please capitalize your texts. It would be better if your paragraphs are capitalized and have no grammar errors 🙏.